_data/retail.yml

```diff
@@ -99,13 +99,13 @@ websites:
       twitter: EBGamesAus
 
     - name: eBay
-      url: http://ebay.com
+      url: https://www.ebay.com
       img: ebay.png
       tfa: Yes
-      hardware: Yes
-      software: Yes
+      sms: Yes
       exceptions:
-          text: "eBay only supports 2FA in some countries."
+          text: "eBay now only supports SMS for 2FA."
```

Comment:
I'm not sure this exception text is necessary since SMS is the only listed option that will be listed on the site, even if `software` and `hardware` used to be supported.

Reply:
Alrighty. I can just delete the exception all together.